.gitignore

```diff
@@ -7,4 +7,5 @@
 /pkg/
 /spec/reports/
 /tmp/
-*.gem
\ No newline at end of file
+*.gem
+/.idea/
```

Comment:
There is no harm at all. Just a little nitpick as we wouldn't want user-specific ignores getting out of hand.